deploy/roles/deploy.builder/templates/operators/qdrouterd/operator.yaml

```diff
@@ -16,7 +16,7 @@ spec:
       containers:
         - name: qdr-operator
           # Replace this with the built image name
-          image: {{ registry_path }}/{{ imagestream_namespace }}/amq-interconnect-operator
+          image: {{ registry_path }}/{{ imagestream_namespace }}/qdr-operator
```

Comment:
:-1: AFAICT this will no longer work with the downstream image streams so we can't merge this. Since this is referring to our local imagestream name, I think we can continue to call it amq-interconnect-operator on our side.

Reply:
Agreed. The image stream names should probably just remain exactly the same as the downstream version. There is probably no reason to change anything other than the `--from=...` part

Reply:
Fixed and tested.